### ISSUE TITLE
Initial Validation Loss hindering ModelCheckpoint and EarlyStopping

### ISSUE THREAD

**Issue by KeironO** [fchollet/keras#1876]
When training Neural Networks I tend to find that the validation loss starts low, increases and then drops down after a few epochs. However, in some cases initial validation loss can be lower than validation loss after training.

Is there a way to ignore the first `x` epochs on callbacks?

**Comment by pasky** [comment]
Just a general suggestion, doesn't this mean your training task is badly formed and your classes are too imbalanced and/or you should actually be using a different loss function?  (You can use your own!)

(I know that it's sometimes not easy, for example for many of my tasks, I have MRR as my evaluation criterium so I have to use a loss that's not exactly the same.)
